### ISSUE TITLE
Warp does not play well with Rocket.app (Slack like Emoji picker app)

### ISSUE THREAD

**Issue by cfarvidson** [warpdotdev/Warp#1927]
### Describe the bug

There is a way to disable an app in [Rocket](https://matthewpalmer.net/rocket/).

But Warp only shows up as `stable`. and the exclution does not work. 

I have not had this problem with any app before, and I think that there is something with how Warp is being built.

### To Reproduce

1. Install Rocket and set `:` to your trigger key
2. Try to add Warp as one of the excluded apps
3. In Warp run `vim somefile.txt`
4. Try to write `:q` to quit

### Expected behaviour

Rocket should not be triggered/activated when trying to quit vim

### Screenshots

<img width="501" alt="Screenshot 2022-10-06 at 13 45 51" src="https://user-images.githubusercontent.com/7823236/194306013-d4beab91-ec1a-4a9f-8a89-1001198e7d30.png">

### Operating System

MacOS

### OS Version

12.6

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin21.0)

### Warp Version

v0.2022.09.27.08.11.stable_00

### Does this block you from using Warp daily?

Yes

**Comment by dannyneira** [comment]
Hi @cfarvidson, Thanks for letting us know! We're sorry about this, this is not an ideal experience. I've been able to repro this but not sure if it's an issue with Rocket not recognizing Warp so I'd advise opening a case with the Rocket developer as well.

**Comment by cfarvidson** [comment]
I have never had this issue with any other app and I’ve been using rocket for years. I really don’t think that the issue is with rocket since it seems that your app is being displayed as “stable”. But I could be wrong!My guess is that it has to do with how you build your app. Some metadata field used to identify the same is being set to “stable”. I'm really not trying to be rude here, so please don’t misunderstand. 🙂I’m not going to file a bug with rocket. You can file a bug with rocket. (I’m just not using warp until it plays better with rocket)I don’t have the time to be the middleman in this support issue. 27 okt. 2022 kl. 23:15 skrev Danny Neira ***@***.***>:﻿
Hi @cfarvidson, Thanks for letting us know! We're sorry about this, this is not an ideal experience. I've been able to repro this but not sure if it's an issue with Rocket not recognizing Warp so I'd advise opening a case with the Rocket developer as well.

**Comment by milch** [comment]
I ran into the same issue, I think it is a problem with how Warp is built. Presumably the fact that the executable is named `stable` is causing some sort of issue, since the convention on macOS is that the executable and the app bundle are using the same name, i.e. `$NAME.app/Contents/MacOS/$NAME`. Anyway, Rocket provides a workaround here: https://matthewpalmer.net/rocket/help.html?ref=rocketinapp#visual-studio-code 

**tl;dr** Add Warp.app as a disabled app through Rocket's preferences. Double click on the generated `stable` entry and rename it to `Warp`.
